src/System.Windows.Forms.Design/src/System/ComponentModel/Design/ByteViewer.cs

```diff
@@ -619,23 +613,14 @@ public virtual void SaveToFile(string path)
             return;
         }
 
-        FileStream currentFile = new FileStream(path, FileMode.OpenOrCreate, FileAccess.ReadWrite, FileShare.None);
-        try
-        {
-            currentFile.Write(_dataBuf, 0, _dataBuf.Length);
-            currentFile.Close();
-        }
-        catch
-        {
-            currentFile.Close();
-            throw;
-        }
+        using FileStream currentFile = new FileStream(path, FileMode.OpenOrCreate, FileAccess.ReadWrite, FileShare.None);
```

Comment:
We prefer to use target typed new:

```suggestion
        using FileStream currentFile = new(path, FileMode.OpenOrCreate, FileAccess.ReadWrite, FileShare.None);
```